partner_delivery_schedule/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_delivery_schedule_user,access_delivery_schedule_user,model_delivery_schedule,base.group_user,1,0,0,0
```

Comment:
I still have doubts with a portal user that accesses the partner if we are going to have permission access. Can you please check?

Reply:
So, a portal user has not access to partner view due to property_delivery_carrier_id field added by delivery module...
Anyway.... a portal user has not access to a partner view (I think), in old versions there was a menu to access to partner but now only access by website templates... so if you do not render the field you can access to a partner view

Reply:
Cleared by direct conversation